packages/strapi-design-system/src/Grid/Grid.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+import styled from 'styled-components';
+
+import { GridContext } from './GridContext';
+import { Box, BoxProps } from '../Box';
+import handleResponsiveValues, { ResponsiveValue } from '../helpers/handleResponsiveValues';
+
+export interface GridProps extends BoxProps {
+  gridCols?: number;
+  gap?: ResponsiveValue;
+}
+
+const GridWrapper = styled(Box)<Required<Pick<GridProps, 'gridCols' | 'gap'>>>`
+  display: grid;
+  grid-template-columns: repeat(${({ gridCols }) => gridCols}, 1fr);
+  ${({ theme, gap }) => handleResponsiveValues('gap', gap, theme)}
+`;
+
+export const Grid: React.FC<GridProps> = ({ gap = '0', gridCols = 12, ...props }) => {
```

Comment:
```suggestion
export const Grid = ({ gap = '0', gridCols = 12, ...props }: GridProps) => {
```

While it's not wrong, we haven't been using FC at the moment (I'm not sure i see the benefits? Happy to be shown though!)

Reply:
IMO it provides better type safety by explicitly indicating that the component is a function component and declaring the props type. It also gives better IntelliSense support in IDEs providing types suggestions like `displayName` and others. Even though, I see you point, if we are not using it maybe we don't want to start now, or do we? Can this be a FE sync topic?

Reply:
> Can this be a FE sync topic 

I think that's a great idea, if you could present the pros / cons etc. I'd be interested to see and we can do the mass change in one PR if we decide to go for it 👍🏼